### PR TITLE
Create ruby.yml

### PR DESCRIPTION
**Pull request by jasylwong**
This PR should be reviewed after the rails setup one.
The (ruby).yml will be needed for setting up CodeClimate as part of issue #3

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Ruby
+    # To automatically get bug fixes and new Ruby versions for ruby/setup-ruby,
+    # change this to (see https://github.com/ruby/setup-ruby#versioning):
+    # uses: ruby/setup-ruby@v1
+      uses: ruby/setup-ruby@ec106b438a1ff6ff109590de34ddc62c540232e0
```
Comment (Stivaros) [inline .github/workflows/ruby.yml]: Why have you chosen to use the commit SHA as a reference here? Genuinely intrigued :)

File: .github/workflows/ruby.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      with:
+        ruby-version: 2.6
```
Comment (Stivaros) [inline .github/workflows/ruby.yml]: At the moment this is using ruby 2.6.6 (see the [build logs here](https://github.com/ruby/setup-ruby)). As per the [documentation](https://github.com/ruby/setup-ruby#supported-version-syntax) I think we can remove this line as it'll try our `.ruby-version` file. One less place to have to update/check these values :)

**Review by Stivaros — CHANGES_REQUESTED** [review]
At the moment this doesn't quite do what we want it to do but well done, this is more than I've done with GH Actions before!

File: .github/workflows/ruby.yml (hunk 2 above)
Comment (jasylwong) [reply .github/workflows/ruby.yml]: The documentation says "short version like 2.6, automatically using the latest release matching that version (2.6.5)".
Doesn't this then match our .ruby-version file which is also 2.6.5?

**Review by Stivaros — APPROVED** [review]
Nice one

**Pull request merged**